### PR TITLE
remove expand when slot is empty

### PR DESCRIPTION
**Pull request by GraceGardner**
## Description & motivation 💭 
Removes hover states, chevron and ability to expand accordion when slot is empty. 

## Testing 🧪 
Storybook 

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
Please checkout the accordion component in storybook 

### Issue(s) closed 
DT-2312

### CODE REVIEW

File: server/go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/temporalio/ui-server/v2
 
-go 1.20
+go 1.21
+
+toolchain go1.22.4
```
Comment (laurakwhit) [inline server/go.mod:5]: Definitely don't want this change in here. Same with changes in `server/go.sum`.

```suggestion
go 1.20
```

**Review by laurakwhit — CHANGES_REQUESTED** [review]

File: src/lib/holocene/accordion.svelte [hunk 2]
```diff
@@ -59,14 +65,16 @@
...
-        <slot name="action" />
-      </div>
-      <Icon class="m-2" name={open ? 'chevron-up' : 'chevron-down'} />
+      {#if $$slots.default}
+        <div
+          class="flex flex-row items-center gap-2 pr-2"
+          on:click|stopPropagation
+          on:keyup|stopPropagation
+        >
+          <slot name="action" />
```
Comment (laurakwhit) [inline src/lib/holocene/accordion.svelte:74]: We may still want the action slot even if there is no default slot (there can be links, etc. in it). 

Maybe we move the `#if $$slots.default` conditional to just be for the `Icon`?

Comment (GraceGardner) [reply src/lib/holocene/accordion.svelte]: That's a good point! Thanks Laura!

File: server/go.mod (hunk 1 above)
Comment (GraceGardner) [reply server/go.mod]: I don't know how I did that. Thanks for catching this.

Comment (laurakwhit) [reply server/go.mod]: @GraceGardner looks like changes to `server/go.sum` still need to be removed.

**Review by laurakwhit — APPROVED** [review]

**Pull request merged**